### ISSUE TITLE
Monster difficulty scores need auditing.

### ISSUE THREAD

**Issue by I-am-Erk** [CleverRaven/Cataclysm-DDA#69707]
### Is your feature request related to a problem? Please describe.

During combat AI testing I have realized that the score we're getting for monsters is pretty wild and inaccurate. This is because there's the expectation that a JSON `difficulty` will be filled out to add some points to monsters that rely on special attacks for their difficulty.

Monsters that just use regular attacks seem to be scored OK

### Solution you would like.

for now we should just go through and check out the reported difficulty of monsters and add some points in JSON if they're too low. Ferals, who rely on their wielded weapons for high damage, are a particularly egregious example.

### Describe alternatives you have considered.

In the long run, some special attacks should be algorithmically added to the difficulty. We'll always need JSON adjustment for special attacks that don't do damage.

### Additional context

This is something that should have a ci test I think. Any creature with multiple special attacks should require a Difficulty entry in the JSON, and we should perhaps test a few low difficulty creatures against a similarly scored NPC to make sure they actually are low difficulty.

**Comment by TheSaddestGoomba** [comment]
The new frog monsters are probably a good example of this. Most notably the foul and electric lines are wildly divergent in rating. I've started looking at it alongside a new PR but would welcome someone else's eyes on it.

**Comment by I-am-Erk** [comment]
Thanks! Yeah the auto generated code gives monsters a buff if they have a lot of special attacks but currently has no way of knowing how bad those attacks might be. Leaping is rated the same as being able to shoot blasts of nuclear energy.

**Comment by PatrikLundell** [comment]
I don't believe the "Good First Issue" label is a good one, unless it's intended to just apply to performing a manual survey of monsters lacking adjustments.

I certainly don't have any inkling about how to set difficulty ratings in a somewhat reasonable way, and I doubt people even greener would either.

**Comment by I-am-Erk** [comment]
To some degree, it's really a pretty easy first issue. You can easily turn on debug reporting to get difficulty reports on monsters and see what ones are being rated much lower than others. "good first issue" doesn't mean the person is a noob to the game, just to contributing.

**Comment by Zireael07** [comment]
If you need to preface it with "to some degree", it's NOT a good first issue

**Comment by LyleSY** [comment]
Perhaps obvious, but I think that the various eating specials should not increase difficulty, or should be handled by a different function like SPECIAL_EAT. Anything else jump out as a problem besides Ferals?
